### ISSUE TITLE
Is it ok to use DiffuScene's objencoder weights

### ISSUE THREAD

**Issue by Boltzmachine** [PhyScene/PhyScene#2]
For objencoder, since the the weigths of PhyScene is not released, and the training script for it is not provided. Is it ok to use DiffuScene's weights?

**Comment by YandanYang** [comment]
Yes, you can have a try. We use the same objencoder as DiffuScene. But I am not sure if the parameter is the same. For example, we take 32 as the feature dimension. You should have a check. And then if you use the 3D feature generated by their weights, you may need to train PhyScene by yourself to fit the 3D feature.

**Comment by YandanYang** [comment]
Hi, the training code for objencoder is released. You can check `DATA.md` for details.
And we will release the weight for objencoder next week.

**Comment by gyula-coder** [comment]
the path 'scripts/train/train_objautoencoder.py' is not exist. please check it.

**Comment by YandanYang** [comment]
I am sorry the training code of autoencoder can not be released right now.
 More solutions can be found in #13
